jaxrs/src/main/java/org/killbill/billing/jaxrs/resources/AccountResource.java

```diff
@@ -688,7 +688,7 @@ public Response payAllInvoices(@PathParam("accountId") final String accountId,
                                            invoice.getBalance() : remainingRequestPayment;
             if (amountToPay.compareTo(BigDecimal.ZERO) > 0) {
                 final UUID paymentMethodId = externalPayment ? null : account.getPaymentMethodId();
-                createPurchaseForInvoice(account, invoice.getId(), amountToPay, paymentMethodId, externalPayment, pluginProperties, callContext);
+                createPurchaseForInvoice(account, invoice.getId(), amountToPay, paymentMethodId, externalPayment, UUIDs.randomUUID().toString(), UUIDs.randomUUID().toString(), pluginProperties, callContext);
```

Comment:
@pierre 
There is only one integration test failing after removing the set.
I've been debugging and found out that [here](https://github.com/javier-gomez-hs/killbill/blob/b31a8ad2e463c5bf9869b4d2c69078e3ea5ecb55/payment/src/main/java/org/killbill/billing/payment/core/sm/control/DefaultControlInitiated.java#L120-L120) is being built the PaymentAttemptModelDao and it is throwing an Exception because the TransactionExternalKey is null.
Should I do something similar to [this](https://github.com/killbill/killbill/blob/master/payment/src/main/java/org/killbill/billing/payment/dao/PaymentTransactionModelDao.java#L60) to avoid this issue?

Thanks!